crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -73,28 +73,28 @@ impl GlobalTransform {
 
     #[doc(hidden)]
     #[inline]
-    pub fn from_translation(translation: Vec3) -> Self {
+    pub const fn from_translation(translation: Vec3) -> Self {
         GlobalTransform {
             translation,
-            ..Default::default()
+            ..Self::identity()
```

Comment:
For my own learning, any reason to prefer `Self::identity()` over `Default::default()`? Is it because `Default` is a trait and won't work here with `const`?

Reply:
exactly, `default()` can't be `const`